app/RedisQueue.py

```diff
@@ -1,94 +0,0 @@
-import redis, uuid, datetime
```

Comment:
File deleted in /app but still available in /whatsapp
https://github.com/teamsempo/SempoBlockchain/blob/master/whatsApp/RedisQueue.py